bazel_tools/scalapb/com/daml/package.proto

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "scalapb/scalapb.proto";
+
+package com.daml;
+
+option (scalapb.options) = {
```

Comment:
not sure if using protobuf to configure your protobuf library is clever or annoying :slightly_smiling_face:

Reply:
Do _not_ get me started. :upside_down_face:

Reply:
next-level dogfooding